src/routes/safe/components/Balances/Coins/index.jsx

```diff
@@ -94,7 +94,7 @@ const Coins = (props: Props) => {
                     <Button
                       className={classes.send}
                       color="primary"
-                      onClick={() => showSendFunds(row.asset.address)}
+                      onClick={showSendFunds(row.asset.address)}
```

Comment:
does it still work? `onClick` would be assigned with value returned by `showSendFunds(row.asset.address)`

Reply:
fixed